rosdep/python.yaml

```diff
@@ -1988,6 +1988,9 @@ python-googleapi:
   fedora: [google-api-python-client]
   gentoo: [dev-python/google-api-python-client]
   ubuntu:
+    bionic: [python-googleapi]
+    bionic_python3: [python3-googleapi]
+    focal_python3: [python3-googleapi]
```

Comment:
This style is deprecated, and we don't use it anymore.  Please make an entirely new rosdep key called `python3-googleapi`, and populate it with these entries.

Reply:
@mqcmd196 Friendly ping. Please move these Python3 packages to a new rosdep key as Chris suggests.

Reply:
sorry,,, I fixed.